### DISCUSSION TITLE
People spectating you count feature?

### DISCUSSION THREAD

**Discussion by scriptkiddyno1** [winstxnhdw/lc-hax#313] [Ideas]
Would this be possible? Would be helpful when doing a bit of trolling.

**Comment by winstxnhdw** [comment]
Hmm, not sure if this is possible.

**Reply by scriptkiddyno1** [reply]
All good! Good job on what you have done so far by the way, and if you don't mind one last suggestion, would making the shotgun go off be possible when someone is holding it?

**Reply by winstxnhdw** [reply]
I don’t think so. The moment someone picks up the gun, they gain ‘ownership’ of it and you can no longer control it.
